lib/broccoli/ember-app.js

```diff
@@ -291,7 +294,11 @@ EmberApp.prototype._initVendorFiles = function() {
     jquery           = ember.paths.jquery;
   } else {
     jquery = this.bowerDirectory + '/jquery/dist/jquery.js';
-    emberShims = this.bowerDirectory + '/ember-cli-shims/app-shims.js';
+
+    if (emberCliShimsFromBower) {
+      emberShims = this.bowerDirectory + '/ember-cli-shims/app-shims.js';
```

Comment:
this is just following what we did before, and I believe that we actually want to guarantee posix style paths (`/`).